pyramid_swagger/ingest.py

```diff
@@ -217,12 +218,13 @@ def create_bravado_core_config(settings):
         'use_models': False,
     }
     configs.update({
-        bravado_core_key: settings[pyramid_swagger_key]
+        bravado_core_key: (settings[pyramid_swagger_key] if bravado_core_key == 'formats'
+                           else asbool(settings[pyramid_swagger_key]))
```

Comment:
This is dangerous, as any new bravado-core config key which doesn't have a boolean value would break this. We should instead list the bravado-core keys we know of and the function we should use for each key to get the value. Ideally we'd provide an API to do this within bravado-core.